plugin/vim-gh-line.vim

```diff
@@ -103,6 +116,27 @@ func! s:gh_line(action) range
     call system(l:finalCmd)
 endfun
 
+func! s:find_git_remote(cdDir)
+  let l:remotes = system(a:cdDir . "git remote")
+  let l:remote_list = split(l:remotes, '\n')
+  let l:remote = ""
+
+  if len(l:remote_list) > 1
+    call inputsave()
+    let l:remote = input('Please select one remote(' . join(l:remote_list, ',') . '): ')
```

Comment:
I guess this is fine to start with. 
But I have expected to use a quickFix window where each remote is listed in a different line. 
The user could make a number selection. Or move up and down in the quickfix list and select visually as well. 

In this implementation, the user needs to type the entire remote name fully. That is also an inconvenience. It would be nicer if we could use smart complete. 
For example if the remotes are called `origin,upstream` it should be ok for the user to type `o` and the plugin could choose a remote with that.

Reply:
this is a good idea! I didn't think about this. I'm not vimscript guru, PR is welcome :)

Reply:
once we merge this PR I can spend time on this on my leisure.